src/test/java/com/activeviam/mac/statistic/memory/scenarios/TestMissingChunkId.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * (C) ActiveViam 2020
+ * ALL RIGHTS RESERVED. This material is the CONFIDENTIAL and PROPRIETARY
+ * property of ActiveViam. Any unauthorized use,
+ * reproduction or transfer of this material is strictly prohibited
+ */
+
+package com.activeviam.mac.statistic.memory.scenarios;
+
+import com.activeviam.fwk.ActiveViamRuntimeException;
+import com.activeviam.mac.cfg.impl.ManagerDescriptionConfig;
+import com.activeviam.mac.memory.MemoryAnalysisDatastoreDescriptionConfig;
+import com.activeviam.mac.statistic.memory.ATestMemoryStatistic;
+import com.activeviam.pivot.utils.ApplicationInTests;
+import com.qfs.desc.IDatastoreSchemaDescription;
+import com.qfs.junit.LocalResourcesExtension;
+import com.qfs.monitoring.statistic.memory.IMemoryStatistic;
+import com.qfs.pivot.monitoring.impl.MemoryStatisticSerializerUtil;
+import com.qfs.store.IDatastore;
+import com.qfs.util.impl.QfsFileTestUtils;
+import com.quartetfs.biz.pivot.IActivePivotManager;
+import com.quartetfs.biz.pivot.definitions.IActivePivotManagerDescription;
+import com.quartetfs.biz.pivot.dto.CellSetDTO;
+import com.quartetfs.biz.pivot.query.impl.MDXQuery;
+import com.quartetfs.fwk.Registry;
+import com.quartetfs.fwk.contributions.impl.ClasspathContributionProvider;
+import com.quartetfs.fwk.query.QueryException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+/** The scenario this test produces has a cube level {@code id} with a nullable dictionary. */
+public class TestMissingChunkId {
+
+  @RegisterExtension protected LocalResourcesExtension resources = new LocalResourcesExtension();
+  private Collection<IMemoryStatistic> memoryStatistics;
+  private final ApplicationInTests analysisApplication = createAnalysisApplication();;
+
+  @BeforeAll
+  public static void setupRegistry() {
+    Registry.setContributionProvider(new ClasspathContributionProvider());
+  }
+
+  @BeforeEach
+  public void setup() throws IOException {
+    final Path statisticsPath =
+            Path.of("src", "test", "resources", "stats_files_with_missing_chunk_id");
```

Comment:
I am not entirely sure that this will work when running the tests with maven.
But at the same time, given that tests should be run from the root of the repo, that contains a single module, it may be.